flixel/system/FlxSound.hx

```diff
@@ -713,6 +726,18 @@ class FlxSound extends FlxBasic
 		}
 		return _time = time;
 	}
+	
+	private function set_loopTime(value:Float):Float
+	{
+		loopTime = FlxMath.bound(value, 0, length);
```

Comment:
Wouldn't we need the inverse check of `set_endTime` here? It doesn't make sense for `loopTime` to be bigger than `endTime`...